src/i2c/src/vhu_i2c.rs

```diff
@@ -76,6 +79,9 @@ struct VirtioI2cOutHdr {
 }
 unsafe impl ByteValued for VirtioI2cOutHdr {}
 
+/// VirtioI2cOutHdr Flags
+const VIRTIO_I2C_FLAGS_M_RD: u32 = 1 << 1;
+
```

Comment:
It's odd we have two styles of bit declarations, one where we expect an explicit shift and one where we do it in the constant.

Reply:
I agree to this, but this looked sane because of the way they are getting used eventually. 

VIRTIO_I2C_F_ZERO_LENGTH_REQUEST is used in features() where we use more (already defined) flags which don't have the shift within them.

While it makes the code more readable to make VIRTIO_I2C_FLAGS_M_RD contain the shift as it is used directly in the code later.